### PR TITLE
Changes for implementing delete guest. #72

### PR DESCRIPTION
**Pull request by mayur43s**
This pull request contains the changes for dashbaord tab #69 and delete guest #72.

### CODE REVIEW

File: Bouncer/Assets.xcassets/Tabbar Icons/sale_tab.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "sale_tab.png",
```
Comment (kidbombay) [inline Bouncer/Assets.xcassets/Tabbar Icons/sale_tab.imageset/Contents.json]: dashboard tab

File: Bouncer/Controllers/SalesViewController.swift [hunk 2]
```diff
@@ -7,24 +7,143 @@
...
+import SwiftyBeaver
+import Kingfisher
 
-
-class SalesViewController: BaseViewController {
-
-
+class SalesViewController: BaseViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    var network: Network!
```
Comment (kidbombay) [inline Bouncer/Controllers/SalesViewController.swift]: I hate how Ben set this up to require these vars in every screen. The API integration was so clean before. Add issue to clean this code up in future builds.

File: Bouncer/Controllers/SalesViewController.swift [hunk 3]
```diff
@@ -7,24 +7,143 @@
...
-
-
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
+    
+    
+    func getAllGuestListForEvent() {
+        
+        // Passing ticketed as nil for fetching all the guestlists from API...
+        network.getEventInvoices(eventID: event.ID!, session: session) { (eventInvoiceResults, apiError) in
```
Comment (kidbombay) [inline Bouncer/Controllers/SalesViewController.swift]: Really should be APIClient.getEventInvoices
or BouncerAPI.getEventInvoices
or UpAllNightAPI.getEventInvoices
save for future

File: Bouncer/Controllers/SalesViewController.swift [hunk 4]
```diff
@@ -7,24 +7,143 @@
...
+        labelSoldQuantity.text = "\(eventInvoice.sold!)/\(eventInvoice.quantity!)"
+        labelCheckedInCount.text = "\(eventInvoice.checkedInCount!)"
+        
+        let sold = eventInvoice.sold!
+        let quantity = eventInvoice.quantity!
+        
+        if quantity == 0 {
+            progressView?.progress = 0.0
+        } else {
+            progressView?.progress = Float(sold)/Float(quantity)
```
Comment (kidbombay) [inline Bouncer/Controllers/SalesViewController.swift]: ProgressView should really have the ability to set this based on max.
ProgressView.max = 100
ProgressView.value = 50
which should result in correct display.

File: Bouncer/Controllers/SalesViewController.swift (hunk 2 above)
Comment (mayur43s) [inline Bouncer/Controllers/SalesViewController.swift]: Sure, I'll add one.

**Pull request merged**